app/ui-react/syndesis/src/modules/data/pages/viewCreate/SelectNamePage.tsx

```diff
@@ -0,0 +1,158 @@
+import { WithViewEditorStates, WithVirtualizationHelpers } from '@syndesis/api';
+import { AutoForm, IFormDefinition } from '@syndesis/auto-form';
+import { RestDataService, SchemaNodeInfo, ViewInfo } from '@syndesis/models';
+import { ViewConfigurationForm, ViewCreateLayout } from '@syndesis/ui';
+import { WithRouteData } from '@syndesis/utils';
+import * as React from 'react';
+import i18n from '../../../../i18n';
+import { PageTitle } from '../../../../shared';
+import resolvers from '../../../resolvers';
+import { ViewCreateSteps } from '../../shared';
+import { generateViewEditorState } from '../../shared/VirtualizationUtils';
+
+export interface ISaveForm {
+  name: string;
+  description?: string;
+}
+
+/**
+ * @param virtualizationId - the ID of the virtualization for the wizard.
+ */
+export interface ISelectNameRouteParams {
+  virtualizationId: string;
+}
+
+/**
+ * @param virtualization - the virtualization for the wizard.
+ * @param schemaNodeInfo - the selected schema node
+ */
+export interface ISelectNameRouteState {
+  virtualization: RestDataService;
+  schemaNodeInfo: SchemaNodeInfo;
+}
+
+export class SelectNamePage extends React.Component {
+  public selectedViews: ViewInfo[] = []; // Maintains list of selected views
+
+  public render() {
+    return (
+      <WithRouteData<ISelectNameRouteParams, ISelectNameRouteState>>
+        {(
+          { virtualizationId },
+          { virtualization, schemaNodeInfo },
+          { history }
+        ) => (
+          // TODO need to retrieve real user here
+          <WithVirtualizationHelpers username="developer">
```

Comment:
This has just been merged, you can get the username from the `AppContext`

Reply:
amended the PR - removed the username property from WithVirtualizationHelpers and now using AppContext in VirtualizationCreatePage to get the username.